### ISSUE TITLE
Bulk create database command

### ISSUE THREAD

**Issue by Ameobea** [fat-whale/tectonicdb#17]

**Comment by 0b01** [comment]
Is this really necessary?

**Comment by Ameobea** [comment]
It takes a good 15 seconds to create all of the databases for a new exchange aggregator we're adding even when running on a local connection, so I figured it would be a nice item for the wish list.  Necessary, no.

**Comment by 0b01** [comment]
How many exchanges? Is this for coinapi?

**Comment by Ameobea** [comment]
Yeah, CoinAPI.  They're listing 14711 individual symbols.

**Comment by 0b01** [comment]
And all the connections are live? That's actually very impressive.

**Comment by Ameobea** [comment]
Yeah, I was impressed.  Their firehose feed was topping out at 7500 insertions/second and 9mbps of network traffic.  Not sure if that's compressed or not.
